### ISSUE TITLE
Widget - Accessibility issues

### ISSUE THREAD

**Issue by jbpenrath** [numerique-gouv/lasuite-integration#48]
Widget component has several accessibility issues :
1. The textarea input has no label.
2. It should structured as a dialog. (`role="dialog"`, `aria-modal="true"` and an aria-label should be set)
3. The `role="status"` is set to a div that is not updated when its content changed. To allow assistive technology to be warn by the content change, this attribute should be put directly on the new content (e.g: On the `p` element in case of success message)
